.github/workflows/release.yml

```diff
@@ -26,25 +26,6 @@ jobs:
           collaborators: 'outside'
         env:
           GITHUB_TOKEN: ${{ secrets.SA_TOKEN }}
-  maintainers:
```

Comment:
This never worked properly! https://github.com/marketplace/actions/contribute-list#keywords